array-manipulation-3/src/equal.js

```diff
@@ -0,0 +1,16 @@
+/* exported equal */
+
+function equal(first, second) {
+  var combinedLength = Math.round(first.length + second.length / 2);
+  var isEqual;
+  var i = 0;
+
+  while (first[i] === second[i] && i < combinedLength) {
+    isEqual = true;
+    i++;
+  }
+  if (i < combinedLength) {
+    isEqual = false;
+  }
+  return isEqual;
```

Comment:
Try this:
- if the arrays are not the same length, return `false`.
- for every index in the first array, if the values in both arrays at that index are not equal, return `false`
- return `true`